lib/utils.py

```diff
@@ -64,7 +64,7 @@ def set_extras_n_fix_units(
     #
     # outbase needs to be an area shared with schedd servers.
     #
-    if args["debug"] > 1:
+    if args["debug"]:
```

Comment:
I think this is actually supposed to be >1, since args.debug is actually a number, and Marc's idea was to make sure that we output that message only if something like `--debug 2` was entered.  Do you think this should not be the case, and that `--debug` should just be on or off (i.e. a `bool`)?

Reply:
I did not change these. I think my merge reverted these debug statements back to a previous version. I will fix them all.

Reply:
OK.  Yeah, there has been a lot of activity in the past few days.

Reply:
done